cluster/hazelcast_cloud_config.go

```diff
@@ -17,12 +17,8 @@
 package cluster
 
 type HazelcastCloudConfig struct {
```

Comment:
Would it make sense to change it to just `CloudConfig` to match with other clients?

I don't have strong opinions about this though, because `Cloud` might be too broad

Reply:
Yeah, I wanted to keep the HazelcastCloud brand there but we can change that to match other clients. The user would access it using `config.Cluster.Cloud` in that case.

Reply:
Renamed that to `CloudConfig` (`config.cluster.Cloud`) at: https://github.com/hazelcast/hazelcast-go-client/pull/585/commits/f3ad108cdecae243f1381d19af33c0723e4d4a96